src/components/RgbaStringColorPicker.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import { AlphaColorPicker } from "./common/AlphaColorPicker";
+import { ColorModel, ColorPickerBaseProps } from "../types";
+import { equalColorString } from "../utils/compare";
+import { rgbaStringToHsva, hsvaToRgbaString } from "../utils/convert";
+
+const colorModel: ColorModel<string> = {
+  defaultColor: "rgb(0, 0, 0, 1)",
```

Comment:
Same as above: `hsvaToRgbaString ` uses `rgba()` format, but the default value here is in `rgb` format.

Reply:
Good catch! Thx!